controllers/clusterversion_controller.go

```diff
@@ -356,3 +369,49 @@ func (c *ClusterVersionReconciler) getOperatorConfig() (*corev1.ConfigMap, error
 	}
 	return cm, nil
 }
+
+func (c *ClusterVersionReconciler) ensureConsolePlugin() error {
+	nginxConf := console.GetNginxConfiguration()
+
+	consoleDeployment := console.GetDeployment(c.OperatorNamespace)
+	err := c.Client.Get(context.TODO(), types.NamespacedName{
+		Name:      consoleDeployment.Name,
+		Namespace: consoleDeployment.Namespace,
+	}, consoleDeployment)
+	if err != nil {
+		return err
+	}
```

Comment:
Why do we need to do Get here?

Reply:
we need to get the UID of the deployment of the console to set the owner of the configmaps and service

odf-console has the console deployment as the owner of these resources. not sure if setting the client-operator as owner is going to have a detrimental affect